### PR TITLE
Video line

### PR DESCRIPTION
**Pull request by nikolaklenje**
-Subscribed Line shows videos from all creators user is subscribed on.
-States update accordingly of which user was clicked on,and displaying them on videoPage

### CODE REVIEW

File: lib/helperFunctions.tsx [hunk 1]
```diff
@@ -5,42 +5,99 @@ import { CreatorContext } from "src/context"
 import { constants } from "./constants"
 import Link from "next/link"
 import { Card } from "src/components"
+const data: Data = require("data/data-videos.json")
+
+
+interface Data {
```
Comment (slavo3dev) [inline lib/helperFunctions.tsx:11]: Those types need to be in separate folder - 
Lib/types

File: lib/helperFunctions.tsx [hunk 2]
```diff
@@ -5,42 +5,99 @@ import { CreatorContext } from "src/context"
...
+  [key: string]: {
+    active: any
```
Comment (slavo3dev) [inline lib/helperFunctions.tsx:13]: I think this is Boolean -

File: lib/helperFunctions.tsx [hunk 3]
```diff
@@ -5,42 +5,99 @@ import { CreatorContext } from "src/context"
...
+      const creatorData = data[creator]
+      if (creatorData.active) {
+        const videos = getVideos(creator)
+        for (const video of videos) {
+          activeVideos.push(video)
+        }
+      }
+    }
+
+    for (let i = activeVideos.length - 1; i > 0; i--) {
```
Comment (slavo3dev) [inline lib/helperFunctions.tsx:57]: I am not sure this will be sustainable on the long run

File: lib/helperFunctions.tsx [hunk 4]
```diff
@@ -5,42 +5,99 @@ import { CreatorContext } from "src/context"
...
+    let activeVideos: VideoType[] = []
 
-  const context = useContext(CreatorContext)
-  function subscribedContent() {
-    context.setState(constants.creatorOne.state)
-    context.setCreator(constants.creatorOne.name)
-    context.setPayment(constants.creatorOne.paymentLink)
-    context.setImage(constants.creatorOne.image)
-    context.setSubscription(constants.creatorOne.subscription)
+    for (const creator in data) {
```
Comment (slavo3dev) [inline lib/helperFunctions.tsx:47]: This should be done with map function

**Review by slavo3dev — CHANGES_REQUESTED** [review]
Small fixes

File: lib/helperFunctions.tsx (hunk 3 above)
Comment (nikolaklenje) [reply lib/helperFunctions.tsx:57]: It would be too slow?

**Review by slavo3dev — APPROVED** [review]

**Review by slavo3dev — APPROVED** [review]

**Pull request merged**